packages/iTwinUI-react/src/core/Dialog/DialogDragContext.tsx

```diff
@@ -0,0 +1,20 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+
+export type DialogDragContextProps = {
+  onPointerDown: (e: React.PointerEvent<HTMLElement>) => void;
+};
+
+export const DialogDragContext = React.createContext<
+  DialogDragContextProps | undefined
+>(undefined);
+
+export const useDialogDragContext = () => {
+  const context = React.useContext(DialogDragContext);
+  return {
+    ...context,
```

Comment:
is this hook even necessary?

Reply:
how is it better than just doing `React.useContext(DialogDragContext)` in those two places?

Reply:
I know that context is always truthy and it is just inside code so more comfortable to use.